### PR TITLE
Check PHP code with PHP_CodeSniffer, fix whitespace issues

### PR DESCRIPTION
**Pull request by kba**
I fixed everything considered to be an error by PSR-2 standards.
Hook in `dist` which can do this automatically before git commit.
Double quoting of vars in shell script to prevent whitespace split.

### CODE REVIEW

**Comment by stweil** [comment]
This pull request might conflict with my latest pull requests (simply because it reformats a lot of code).

**Comment by kba** [comment]
#13 needs a bit of intervention but not much. No conflicts beyond that.

File: isbn/lib.php [hunk 1]
```diff
@@ -69,7 +69,8 @@
     'produktSigel' => '//feld[@nr="078" and @ind="e"]/uf[@code="a"]',
 );
 
-function performMapping($map, $outputXml) {
+function performMapping($map, $outputXml)
+{
```
Comment (zuphilip) [inline isbn/lib.php]: Are the parantheses in functions and loops handled differently in this coding style?

**Comment by zuphilip** [comment]
Note to myself: The differences except the whitespaces can be seen in github with the `&w=1` option, e.g. https://github.com/kba/malibu/commit/33bf6eb8bb0e62c760ecfd111f651a1c1477f94c?w=1

File: isbn/lib.php (hunk 1 above)
Comment (stweil) [inline isbn/lib.php]: Yes, PSR-2 puts them on a new line for functions. See http://www.php-fig.org/psr/psr-2/, chapter 4.3.

Comment (zuphilip) [inline isbn/lib.php]: Okay. I guess I am not so much used to that... I think JavaScript is doing that differently...

**Comment by zuphilip** [comment]
I merged also my branch `swiss` today with some new files. There could be some more issues in these files, i.e. code which does not confirm with the standard...

**Comment by zuphilip** [comment]
Thank you very much, @kba . I am looking forward for your next PR with the travis file. :octopus:

**Pull request merged**